kythe/go/extractors/config/runextractor/constants/constants.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package constants defines relevant constants common to multiple extractors.
+package constants
+
+import ()
+
+func requiredEnvVars() []string {
```

Comment:
This probably does not need to be a function. I think it could more simply be:

```
var (
	requiredEnv     = []string{"KYTHE_CORPUS", "KYTHE_ROOT_DIRECTORY", "KYTHE_OUTPUT_DIRECTORY"}
	requiredJavaEnv = append(requiredEnv, "JAVAC_EXTRACTOR_JAR", "REAL_JAVAC")
)
```

(exported or not doesn't matter, the same thing works either way—static initialization order is well-defined)

Reply:
done